srcpkgs/radicale/template

```diff
@@ -1,34 +1,51 @@
 # Template file for 'radicale'
 pkgname=radicale
-version=1.1.6
-revision=4
-archs=noarch
+version=3.0.6
+revision=1
 wrksrc="Radicale-${version}"
 build_style=python3-module
-pycompile_module="radicale"
 hostmakedepends="python3 python3-setuptools"
-depends="python3-pam python3-requests python3-dulwich"
+depends="python3-pam python3-requests python3-dulwich python3-vobject
+ python3-dateutil python3-passlib python3-bcrypt python3-defusedxml
+ apache-htpasswd"
 short_desc="Complete calendar and contact storing and manipulating solution"
-maintainer="lemmi <lemmi@nerd2nerd.org>"
+maintainer="teldra <teldra@rotce.de>"
 license="GPL-3.0-or-later"
 homepage="https://radicale.org"
-distfiles="${PYPI_SITE}/R/Radicale/Radicale-${version}.tar.gz"
-checksum=c007198ea45ef797344672c681d4c13f8b4aa85c15c41a1156225767a405c92b
-
+changelog="https://github.com/Kozea/Radicale/blob/${version}/NEWS.md"
+distfiles="https://github.com/Kozea/Radicale/archive/${version}.tar.gz"
+checksum=69cfbcc66f8dc13fb82c82ed9561a9aad30462a6bb09b45a64ad39401bf61b98
+provides="radicale-${version}_${revision}"
 conf_files="
  /etc/radicale/config
+ /etc/radicale/rights
+ /etc/radicale/users
  /etc/sv/radicale/log/run"
 make_dirs="
  /etc/radicale 755 root root
  /usr/share/radicale/ 755 root root
  /var/log/radicale/ 700 root root
- /var/lib/radicale 750 radicale radicale"
-system_accounts="radicale"
-radicale_homedir="/var/lib/radicale"
+ /var/lib/radicale 750 _radicale _radicale"
+system_accounts="_radicale"
```

Comment:
There is no real utility in changing the system account. Users of the earlier `radicale` and `radicale2` packages will already have a `radicale` system account; this would pollute the account database and sow confusion.

Reply:
I made the proposal to unify the naming of systemaccounts. I stay with `radicale`.